scripts/merge-script/console.ts

```diff
@@ -0,0 +1,55 @@
+import chalk from 'chalk';
+import {prompt} from 'inquirer';
+import {PullRequest, PullRequestFailure, PullRequestStatus} from './pull-request';
+
+/** Prompts the user with a confirmation question and a specified message. */
+export async function promptConfirm(message: string, defaultValue = false): Promise<boolean> {
+  return (await prompt<{result: boolean}>({
+           type: 'confirm',
+           name: 'result',
+           message: message,
+           default: defaultValue,
+         }))
+      .result;
+}
+
+/** Prints a list of pull request failures. */
+export function printPullRequestsWithErrors(failures: PullRequestFailure[]) {
+  failures.forEach(failure => {
+    console.error(
+        chalk.yellow(`  • #${failure.prNumber} - ${getPullRequestFailureMessage(failure)}`));
+  });
+}
+
+/** Prints a list of merged pull requests. */
+export function printMergedPullRequests(merged: PullRequest[]) {
+  merged.forEach(({prNumber, targetBranches}) => {
+    console.error(chalk.yellow(`  • #${prNumber} - Merged into: ${targetBranches.join(', ')}`));
+  });
+}
+
+/** Gets a human-readable message from a pull request failure. */
+function getPullRequestFailureMessage({status, context}: PullRequestFailure) {
+  switch (status) {
+    case PullRequestStatus.CLA_UNSIGNED:
+      return `CLA has not been signed. Please make sure the PR author has signed the CLA.`;
+    case PullRequestStatus.FAILING_CI_JOBS:
+      return `Failing CI jobs.`;
+    case PullRequestStatus.PENDING_CI_JOBS:
+      return `Pending CI jobs.`;
+    case PullRequestStatus.NO_TARGET_LABEL:
+      return `No target branch could be determined. Please ensure a target label is set.`;
+    case PullRequestStatus.NOT_MERGE_READY:
+      return `Not marked as merge ready.`;
+    case PullRequestStatus.CHERRY_PICK_CONFLICTS:
+      return `Could not cherry-pick pull request into branch: ${context}`;
```

Comment:
Probably want to use the phrase "merge conflict" in this message; the current wording omits the reason the change can't be cherry-picked